src/main/java/org/prebid/server/bidder/brightroll/BrightrollBidder.java

```diff
@@ -27,11 +27,11 @@
 import org.prebid.server.proto.openrtb.ext.ExtPrebid;
 import org.prebid.server.proto.openrtb.ext.request.brightroll.ExtImpBrightroll;
 import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.spring.config.bidder.model.BidderAccount;
```

Comment:
Since `BidderAccount` is intended for use exclusively within Spring configuration and wiring code it shouldn't be here. Please create another holder class in `org.prebid.server.bidder.brightroll.model` package that will be used by the bidder implementation.

Reply:
Thanks, good point to separate logic